CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_zendesk v0.11.0
+[PR #98](https://github.com/fivetran/dbt_zendesk/pull/98) includes the following updates:
+## Feature Updates:
+- Added support of the new `schedule_holiday` table in the `schedule_spine` intermediate model in order to properly capture how holidays impact ticket schedules and their respective SLAs.
+- Made relevant downstream changes within the following models to capture proper business hour metrics when taking into account holiday schedules:
+  - `int_zendesk__agent_work_time_business_hours`
+  - `int_zendesk__reply_time_business_hours`
+  - `int_zendesk__reply_time_combined`
+  - `int_zendesk__requester_wait_time_business_hours`
```

Comment:
these feed into other end models right? should we note the the downstream models whose data will be affeceted?

Reply:
Yeah that is a good call. Will update.

Reply:
Ah actually they only feed into the `zendesk__sla_policies`. So this list is all inclusive. Here is the validation using our docs.

https://fivetran.github.io/dbt_zendesk/#!/overview?g_v=1&g_i=int_zendesk__agent_work_time_business_hours%2B%20int_zendesk__reply_time_business_hours%2B%20int_zendesk__reply_time_combined%2B%20int_zendesk__requester_wait_time_business_hours%2B